contact.html

```diff
@@ -0,0 +1,120 @@
+            <div class="d-flex align-items-center">
+                <img src="/assets/moi.jpg" class="p-pic" alt="photo de profil">
+                <div>
+                    <h2 class="fw-bolder padding-h2">27ans, future développeur et chef de profession</h2>
+                    <div class="bar-noir"></div>
+
+                </div>
+            </div>
+
+
+                    <h2 class="fw-bolder padding-h2">Ville de Québec</h2>
+                    <div class="bar-noir"></div>
+
+                    <img class="map" src="/assets/map.png" alt="Carte du Québec">
+                    <p>Québec, QC - Canada</p>
+
+                    <h2 class="fw-bolder padding-h2">Autres liens</h2>
+                    <div class="bar-noir"></div>
+
+                    <ul class="margin-contact">
+                        <li class="padding-projets bold"><img class="logo-projets-garneau-contact"
+                            src="/assets/gitlab.svg" alt="gitlab logo"><a class="link" href="https://gitlab.com/AntoineBergeron720">AntoineBergeron720</a></li>
+                        <li class="padding-propos bold"><img class="logo-projets-garneau-contact"
+                            src="/assets/github.png" alt="github logo"><a class="link" href="https://github.com/AntoineBergeron720">AntoineBergeron720</a></li>
+                        <li class="padding-projets-last bold"><img class="logo-projets-garneau-contact-outlook"
+                            src="/assets/outlook.svg" alt="outlook logo"><a class="link" href="https://outlook.com">antoine.bergeron1@hotmail.com</a></li>
+                        <li class="padding-projets-last bold"><img class="logo-projets-garneau-contact-teams"
+                            src="/assets/teams.png" alt="teams logo"><a class="link" href="https://teams.microsoft.com/dl/launcher/launcher.html?url=%2F_%23%2Fl%2Fentity%2Fd3d1be68-066c-4967-a74b-9edcf902dcfb%2FteamsDashboard%3Fcontext%3D%257B%2522subEntityId%2522%3A%2522SourceData-%2Credirected-from%3D%253EAD_Google_PROSPECTS_All-Other-Countries__Brand-Microsoft_RSA%2522%257D%26src%3Dadwords%26utm_campaign%3DBrand_Microsoft_AllCountries%26utm_medium%3Dppc%26utm_source%3Dadwords%26utm_term%3Dmicrosoft%2520teams%2520meeting%2520download%26hsa_net%3Dadwords%26hsa_ad%3D604265344766%26hsa_kw%3Dmicrosoft%2520teams%2520meeting%2520download%26hsa_mt%3Db%26hsa_src%3Dg%26hsa_cam%3D15060775870%26hsa_ver%3D3%26hsa_acc%3D4570815156%26hsa_tgt%3Dkwd-916834033154%26hsa_grp%3D136463789903%26gad%3D1%26gclid%3DCjwKCAjw0ZiiBhBKEiwA4PT9z2KvyqlDR8UL15gG-vEI3azi-JNBj_BG8u3BzFgd7P4jTqMrKPWgpRoCCBUQAvD_BwE&type=entity&deeplinkId=d5f4f576-067b-4a1e-bf51-b56bf14be2e2&directDl=true&msLaunch=true&enableMobilePage=true&suppressPrompt=true">Antoine Bergeron (anber011)</a></li>
+        
+                    </ul>
+                    <div class="bar-noir">
+                        <a href="#"><img class="logo-arrow" src="/assets/arrow.png" alt="flèche vers le haut"></a>
+
+                      
```

Comment:
Empty blank lines. You can remove them and also format your page to better indentation.